packages/lib/src/utils/geocoder.ts

```diff
@@ -1,3 +1,6 @@
+import { MapRef } from "react-map-gl";
+import { Feature, options } from "../../../components-lib/src/hooks/useGeocoder";
```

Comment:
we usually add an empty line after external library imports and internal ones

Reply:
[commit](https://github.com/Probable-Futures/apps-public/pull/1/commits/0182cd894c4aa3bcdd852408ce9d0e7e33b97ce7)